### ISSUE TITLE
Recommendation on picture links, possible future obsolete links.

### ISSUE THREAD

**Issue by TheFern2** [aspittel/coding-cheat-sheets#5]
On stacks and queue I've noticed the URL links for the images are actual sites links. I recommend creating an images folder and download the pictures, and correct URL to local repo directory. We run into the issues of sites being taking down, or even URL paths get restructured from time to time. Thus making image links obsolete.

**Comment by aspittel** [comment]
yeah -- I could create my own images in Sketch as well, that would probably be the best solution so I don't run into IP issues!
